src/gsy_e/setup/gsy_e_settings.json

```diff
@@ -201,7 +201,7 @@
       "FINAL_SELLING_RATE": 0,
       "MAX_WIND_TURBINE_OUTPUT_W": 160
     },
-    "IAASettings": {
+    "MASettings": {
```

Comment:
This suggests that you did not run `fab write_default_settings_file` which updates the default settings file automatically. AFAIU, also the reason for the "bug" here: https://github.com/gridsingularity/gsy-e/pull/1307/files#r753275470

Reply:
I didn't run it, but I'm not sure that this would have solved the issue. I ran it now and it didn't change the lines that I manually edited.
BTW: as I ran it, it added new lines (e.g. for the PV azimuth settings etc). I suppose that this command is not used that often, and it's not described in our internal wiki. When should we run it?

Reply:
`it's not described in our internal wiki.` --> indeed. This we reminded ourselves in the PRs if somebody forgot to run it as soon as the ConstSettings  or GlobalConfig changed. (also the answer to `When should we run it`) 

I am not sure how to handle this, this can indeed be part of our workflow. (Fortunately, nobody is using this feature AFAIK. Users normally change the Settings in the setup file directly. So this is not breaking stuff)

Reply:
I've done it in d993287.

Reply:
Late to the discussion here, however I think that it is a good time to add it as a part of fab sync in order to be generated automatically on every sync and to not forget it. No need to do it now, I think that a followup ticket would be more appropriate.